sympy/core/relational.py

```diff
@@ -180,13 +206,24 @@ def canonical(self):
         """
         args = self.args
         r = self
-        if r.rhs.is_Number:
-            if r.lhs.is_Number and r.lhs > r.rhs:
+        if r.rhs.is_number:
```

Comment:
You might want to relax this to `not r.rhs.free_symbols` since this will allow `f(1) < x` to then be written as `x > f(1)`.

Reply:
Makes sense I think. Unless it then becomes `-x < -f(1)`, depending on what is decided. I realize that cases like that is probably the reason for selecting the way I did.